src/tutorials/narratives-how-to-write.rst

```diff
@@ -0,0 +1,345 @@
+===================
+Writing a narrative
+===================
+
+The following will walk you through writing a Nextstrain narrative.
+We'll start with a basic 2-slide narrative and then gradually add further slides with the intention
+of describing the functionality of narratives. The aim is for you to feel comfortable with
+the technical aspects of creating and troubleshooting narratives.
+
+
+For an introduction to the concept of Narratives, please see `Communicating Results Using Narratives </guides/communicate/narratives-intro.html>`__ or take a look at this example narrative,  `Twenty years of West Nile virus <https://next.nextstrain.org/narratives/twenty-years-of-WNV>`__.
+
+This tutorial will rely on the new `Narratives Debugger <https://dev.nextstrain.org/edit/narratives>`__ which allows you to drag & drop the markdown files we'll introduce here and see an instant summary of the narrative.
```

Comment:
I think we should consistently capitalize Markdown as the proper noun it is.

```suggestion
This tutorial will rely on the new `Narratives Debugger <https://dev.nextstrain.org/edit/narratives>`__ which allows you to drag & drop the Markdown files we'll introduce here and see an instant summary of the narrative.
```

There are several examples of "markdown" in other places.

Reply:
Fixed - https://github.com/nextstrain/docs.nextstrain.org/compare/03a1276b5dfebbb8d81a3447d64cd31239b9f4e4..477e27d73e3cde37e40c662e0b3aea0a9c4cb987